### PR TITLE
[TDL-38] Display invoice state & summary

### PR DESCRIPTION
**Pull request by ardier16**
Dependent on https://github.com/tokend/web-client/pull/223

### CODE REVIEW

File: src/vue/modules/loyalty-points/create-invoice-form/components/invoice-state-viewer.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="incoming-state-viewer">
+    <p
+      v-if="invoice.isSuccessful"
+      class="invoice-state-viewer__state--successful"
```
Comment (max-nes) [inline src/vue/modules/loyalty-points/create-invoice-form/components/invoice-state-viewer.vue]: invalid use of modifier

File: src/vue/modules/loyalty-points/create-invoice-form/components/invoice-state-viewer.vue [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    >
+      <i class="mdi mdi-check-circle" />
+      <span>
+        {{ 'create-invoice-form.payment-successful-state' | globalize }}
+      </span>
+    </p>
+
+    <p
+      v-else-if="invoice.isPending"
+      class="invoice-state-viewer__state--pending"
```
Comment (max-nes) [inline src/vue/modules/loyalty-points/create-invoice-form/components/invoice-state-viewer.vue]: same here

File: src/vue/modules/loyalty-points/create-invoice-form/components/invoice-viewer.vue [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      {{ 'create-invoice-form.qr-code-description' | globalize }}
+    </p>
+
+    <div class="invoice-viewer__qr-code">
+      <qr-code
+        :text="invoiceQrCodeValue"
+        :margin="0"
+        :size="250"
+        :color-light="'#f2f2f4'"
+        :color-dark="'#262626'"
```
Comment (max-nes) [inline src/vue/modules/loyalty-points/create-invoice-form/components/invoice-viewer.vue]: useless `:`

Comment (ardier16) [reply src/vue/modules/loyalty-points/create-invoice-form/components/invoice-viewer.vue]: Removed

File: src/vue/modules/loyalty-points/create-invoice-form/components/invoice-state-viewer.vue (hunk 2 above)
Comment (ardier16) [reply src/vue/modules/loyalty-points/create-invoice-form/components/invoice-state-viewer.vue]: Updated

File: src/vue/modules/loyalty-points/create-invoice-form/components/invoice-state-viewer.vue (hunk 1 above)
Comment (ardier16) [reply src/vue/modules/loyalty-points/create-invoice-form/components/invoice-state-viewer.vue]: Updated

**Pull request merged**